CHANGES.rst

```diff
@@ -1,3 +1,12 @@
+11.17.01 (2023-05-02)
```

Comment:
As per the other PR, leave the date as "unreleased". The release process will fill in dates automatically.

Reply:
Ah thanks for that clarification, I'll use that in the other PR.